paddle/fluid/inference/api/mkldnn_quantizer.cc

```diff
@@ -138,7 +143,10 @@ void AnalysisPredictor::MkldnnQuantizer::CalculateSingleScale(
       scales_[var_name] = GetMaxScalingFactor(var_tensor, is_unsigned);
       break;
     case ScaleAlgo::MAX_CH:
```

Comment:
What is the difference between `ScaleAlgo::MAX_CH` and `ScaleAlgo::MAX_CH_T`? Could you please add some comments here?